app/views/layouts/blacklight/base.html.erb

```diff
@@ -39,7 +39,7 @@
 <%= render partial: 'shared/announcement' if Psulcat::Application.config.show_announcement %>
 <%= render partial: 'shared/header_navbar' %>
 
-<div id="main-container" class="<%= container_classes %>">
+<div id="main-container" class="blacklight-main-container <%= container_classes %>">
```

Comment:
@cdmo just trying to understand, what is `blacklight-main-container` for?

Reply:
Good question. It's used in [_mobile.scss](https://github.com/psu-libraries/psulib_blacklight/blob/ab0a9cfc8a6692cd01aa933ec9b583ca178a7136/app/javascript/psulib_blacklight/styles/_mobile.scss#L34-L37)

Just to add margin to the top and bottom at smaller breakpoints because otherwise the container slams up into the header.

I mistakenly thought in a previous commit that it was only applying to the main central container, but it applies to the navbar container and search form container too so it was creating extra spacing where not intended and i dont know how I missed it before. See areas in yellow boxes

![Screenshot 2019-08-22 09 54 58](https://user-images.githubusercontent.com/523381/63520595-f6832480-c4c2-11e9-83b1-b40a491456e8.png)

Reply:
Oh and I generally avoid using `id`s in CSS as a rule. Here's [a good post on why](http://oli.jp/2011/ids/)

Main reason is brittle-ness and just the general feeling that `id`s are redundant in CSS. Honestly, I don't like them for anything. lol